### ISSUE TITLE
Fatal error: Class 'Timber' not found in /var/www/html/wp-content/themes/

### ISSUE THREAD

**Issue by dangthanhmiu** [timber/timber#1308]
Hi!

I installed wordpress version 4.7.1 and timber via composer (composer require  wpackagist-plugin/timber-library) and I got this error, which I have no idea how to solve it. Please help

`Fatal error: Class 'Timber' not found in /var/www/html/wp-content/themes/`

**Comment by timknight** [comment]
@dangthanhmiu I've had issues with the wpackagist-plugin install before. Maybe give the instructions at http://timber.github.io/timber/#installation and try to use `composer require timber/timber`. I just used that to create a three file (style.css, blank index.php, functions.php) test theme and it worked just fine.

**Comment by dangthanhmiu** [comment]
@timknight Thank for your reply. I have tried to install via `composer require timber/timber` but the plugin does not installed in plugins folder, so if i use `wpackagist-plugin/timber-library` then, timber is fetched to plugins folder and after that i got this error from log file:

` WordPress database error Table 'sitedb.wp_timber_errorpaths' doesn't exist for query SHOW FULL COLUMNS FROM `wp_timber_errorpaths` made by Timber->fatal_error_handler, Timber->error_handler`

**Comment by timknight** [comment]
Okay @dangthanhmiu I understand what you mean as far as why you went with `wpackagist-plugin/timber-library` instead. And once it was installed in the plugins folder you also went into wp-admin and activated it as well? The activation should then create the needed tables for the plugin.

**Comment by jarednova** [comment]
@dangthanhmiu it appears that it's installing this (unrelated) version of Timber:

https://wordpress.org/plugins/timber/

... there are no DB modifications with _real_ Timber. Make sure that when installing from WPackagist you're using `wpackagist-plugin/timber-library` and not `wpackagist-plugin/timber`

**Comment by dangthanhmiu** [comment]
I used `wpackagist-plugin/timber-library` and after that i am able to login so I cannot active timber.

**Comment by timknight** [comment]
@dangthanhmiu you should be able to do `composer remove wpackagist-plugin/timber-library` and then do `composer require wpackagist-plugin/timber` like @jarednova pointed out.

**Comment by dangthanhmiu** [comment]
@timknight. the `wpackagist-plugin/timber-library` is the one i used and also @jarednova recommend.

**Comment by timknight** [comment]
My apologizes I read that incorrectly.

I would just make sure that you don't have any code in your functions.php creating a Timber instance yet. Or do something like this in your functions.php file which should capture the class error if it's not activated yet and let you get into the admin to activate it.

```
if ( ! class_exists( 'Timber' ) ) {
    add_action( 'admin_notices', function() {
        echo '<div class="error"><p>Timber not activated. Make sure you activate the plugin in <a href="' . esc_url( admin_url( 'plugins.php#timber' ) ) . '">' . esc_url( admin_url( 'plugins.php' ) ) . '</a></p></div>';
    } );
    return;
}
```

**Comment by dangthanhmiu** [comment]
@timknight I have already that code in my functions.php :/

**Comment by dangthanhmiu** [comment]
Just found a solution by deactivate all plugins and then install Timber via Wordpress plugins, not via composer.

**Comment by gsalkin** [comment]
I'm having this issue when installing the update via the plugin updater.  I've removed composer from the equation but it spits out these errors: 

`/content/plugins/timber-library/lib/Timber.php): failed to open stream: No such file or directory in /var/www/my-site/public/content/vendor/composer/ClassLoader.php on line 412`

Anyone have an idea?  I have recently refactored all my code to support Timber 1.x (up from 0.22.6) and it works perfectly on my local VM.
